OpenRA.Mods.Common/Traits/Render/WithParachute.cs

```diff
@@ -32,7 +32,8 @@ public class WithParachuteInfo : UpgradableTraitInfo, ITraitInfo, IRenderActorPr
 		[SequenceReference("Image")] public readonly string ClosingSequence = null;
 
 		[Desc("Palette used to render the parachute.")]
-		public readonly string Palette = "player";
+		[PaletteReference("IsPlayerPalette")] public readonly string Palette = "player";
```

Comment:
Could you add a newline after this string definition (if you insist on putting the attribute on the same line).

This could confuse someone in the future because typically attributes are placed on the line before the object they apply to.